channels/smartcard/client/smartcard_operations.c

```diff
@@ -660,16 +660,40 @@ static LONG smartcard_GetStatusChangeA_Decode(SMARTCARD_DEVICE* smartcard,
 	return status;
 }
 
+
+#define CARD_GET_STATUS_CHANGE(call, operation, ret, status, status_change)			\
```

Comment:
Better use a static inline function for this.

Reply:
Indeed, there are several modules, but they make sense only when used in combination.  
Is it ok for you if I split the PR module by module, even if they remain dead-code until everything is merged?